libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
Ideally we'd use the 'indent with tabs and align with spaces' approach uniformly (which is immune to tab width I believe).

Unfortunately, Xcode happily indents and aligns, but then normalizes everything to tabs then spaces (if the number of spaces is less than the current tab width) - which is incorrect :(

I suspect the only solution here will be a wee tool which can be run on code locally to correct alignment before submitting a patch. Basically, alignment cannot be fixed manually without editing all the modified lines, by hand, in a text editor which preserves tabs and spaces as they are.